packages/sit-onyx/src/components/OnyxNavBar/OnyxNavBar.ct.tsx

```diff
@@ -452,6 +453,32 @@ test("should display More Items correctly", async ({ mount, page }) => {
   });
 });
 
+test("should work with async nav items", async ({ mount, page }) => {
+  // ARRANGE
+  await page.setViewportSize({ width: ONYX_BREAKPOINTS.sm, height: 400 }); // set small viewport, so that the last element ends up in the more menu
+  const component = await mount(AsyncTestCase);
+  await expect(component.getByRole("menuitem").count()).resolves.toBe(0);
```

Comment:
```suggestion
  await expect(component.getByRole("menuitem")).toHaveCount(0);
```

Same for the other expect below